app/src/main/res/layout/main_fragment.xml

```diff
@@ -14,72 +14,67 @@
-            android:layout_height="629dp"
-            android:layout_marginTop="86dp"
-            android:layout_marginEnd="6dp"
-            android:layout_marginRight="6dp"
-            android:layout_marginBottom="16dp"
-            app:layout_constraintTop_toBottomOf="@+id/rcpSearch"/>
-
-        <Button
-            android:id="@+id/btnShoppingList"
-            android:layout_width="100dp"
-            android:layout_height="wrap_content"
-            android:layout_marginStart="10dp"
-            android:layout_marginLeft="10dp"
-            android:text="Shopping List"
-            android:textSize="9dp"
-            app:layout_constraintStart_toStartOf="parent"
-            tools:layout_editor_absoluteY="105dp" />
-
-        <Button
-            android:id="@+id/btnCatagories"
-            android:layout_width="100dp"
-            android:layout_height="wrap_content"
-            android:layout_marginStart="40dp"
-            android:layout_marginLeft="40dp"
-            android:text="Catagories"
-            android:textSize="9dp"
-            app:layout_constraintStart_toEndOf="@+id/btnShoppingList"
-            tools:layout_editor_absoluteY="105dp" />
-
-        <Button
-            android:id="@+id/btnSchedule"
-            android:layout_width="100dp"
-            android:layout_height="wrap_content"
-            android:layout_marginStart="40dp"
-            android:layout_marginLeft="40dp"
-            android:layout_marginEnd="10dp"
-            android:layout_marginRight="10dp"
-            android:text="Schedule"
-            android:textSize="9dp"
+            android:layout_width="wrap_content"
+            android:layout_height="0dp"
+            android:layout_marginTop="8dp"
+            app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintHorizontal_bias="0.166"
-            app:layout_constraintStart_toEndOf="@+id/btnCatagories"
-            tools:layout_editor_absoluteY="105dp" />
+            app:layout_constraintHorizontal_bias="1.0"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/txtShowing"
+            app:layout_constraintVertical_bias="0.0" />
 
         <TextView
             android:id="@+id/txtShowing"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginTop="10dp"
-            android:layout_marginBottom="10dp"
-            android:text="Showing:    Recipes"
-            app:layout_constraintBottom_toTopOf="@+id/lstRecipes"
-            app:layout_constraintTop_toBottomOf="@+id/btnCatagories"
+            android:layout_marginTop="8dp"
             android:gravity="center_horizontal"
-             />
+            android:text="@string/selectaRecipe"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/linearLayout" />
```

Comment:
Re-designed the layout and solved the errors.
Source:
https://www.youtube.com/watch?v=vXjJyLpydxg